### ISSUE TITLE
Get rid of weird typeof syntax

### ISSUE THREAD

**Issue by LeaVerou** [mavoweb/mavo#730]
Until we can merge groups and primitives (#590), we should get rid of this weird `typeof` syntax, rewrite existing `typeof` to `mv-group` (possibly with a warning) and only mention `mv-group` in docs.

Thoughts, @karger @DmitrySharabin?

**Comment by DmitrySharabin** [comment]
Is there a chance that someone uses `typeof` inside their app but not as Mavo’s attribute denoting groups but to intentionally add semantics to their page? And if we replace those `typeof`s with `mv-group`s will we break the semantics of a user’s page?

Anyway, I like the idea of making Mavo consistent in naming its attributes. So I vote for getting rid of `typeof`. I wonder will this process lead us to renaming `property` to something more Mavo-like in the near future? :)

**Comment by LeaVerou** [comment]
> Is there a chance that someone uses `typeof` inside their app but not as Mavo’s attribute denoting groups but to intentionally add semantics to their page? And if we replace those `typeof`s with `mv-group`s will we break the semantics of a user’s page?

Yes, we should not remove `typeof`, only add `mv-group` *as well*.

> Anyway, I like the idea of making Mavo consistent in naming its attributes. So I vote for getting rid of `typeof`. I wonder will this process lead us to renaming `property` to something more Mavo-like in the near future? :)

There was a thread on that somewhere, but I can't find it now.

**Comment by DmitrySharabin** [comment]
I believe you are taking about this one?

https://twitter.com/mavoweb/status/1006605015333687296?s=21
